README.md

```diff
@@ -26,9 +26,13 @@ npm install
 ```sh
 npm start
 
-# Or use node to run bot directly
+# Or use node to run bot directly (puppeteer mode)
 
 node examples/starter-bot.js
+
+# Or start bot with padchat mode for a quick play (recommended)
+
+WECHATY_PUPPET_PADCHAT_TOKEN=botorange_tempory_token WECHATY_PUPPET=padchat node examples/starter-bot.js
```

Comment:
Maybe should change to:

- Get WECHATY_PUPPET_PADCHAT_TOKEN from https://github.com/chatie/wechaty/issues/1296

- Set WECHATY_PUPPET_PADCHAT_TOKEN and Set WECHATY_PUPPET=padchat

Reply:
Yes, you are right.

How about change this 
```
WECHATY_PUPPET_PADCHAT_TOKEN=botorange_tempory_token WECHATY_PUPPET=padchat node examples/starter-bot.js
```

to 
```
WECHATY_PUPPET_PADCHAT_TOKEN=botorange_tempory_token WECHATY_PUPPET=padchat node examples/starter-bot.js

Get token from [Chatie/wechaty#1296](https://github.com/Chatie/wechaty/issues/1296)

See more in [Wiki](https://github.com/lijiarui/wechaty-puppet-padchat/wiki)
```